app/views/errors/error_422.html.erb

```diff
@@ -0,0 +1,6 @@
+<% content_for(:title, 'The change you wanted was rejected') %>
+
+<h1 class="page-title">
+  <%= @custom_header || 'The change you wanted was rejected' %>
+</h1>
+<p><%= @custom_body || 'Your link has expired, or your email does not exist in our database' %></p>
```

Comment:
I think user's are seeing this after clicking the confirm button on `/email/manage/authenticate` and `/email/subscriptions/new` page, not after clicking a link in their email so this error message is potentially confusing.

We catch and manage errors from missing email addresses, so I don't think user will see this page because a that.
Think this page is shown due to invalid CSFR tokens, either old cookies persisting or CSRF token in the HTML might be cached.